CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ Pull requests SHALL be the best way to propose changes to the codebase.
 
 ## Documentation
 
-All documentation MUST conform to our [Documentation Guide](https://github.com/StanfordSpezi/.github/blob/main/DocumentationGuide.md)
+All documentation MUST conform to our [Documentation Guide](https://github.com/StanfordSpezi/.github/blob/main/DOCUMENTATIONGUIDE.md)
```

Comment:
Good catch here! This should actually ideally link to our Spezi DocC documentation, the documentation guide is in there and the md file only links to it. The md rendering is way nicer there 👍

Reply:
I've updated this to link to: https://swiftpackageindex.com/stanfordspezi/spezi/0.7.3/documentation/spezi/documentation-guide